bin/install_grass.sh

```diff
@@ -54,7 +54,7 @@ TMP_DIR=/tmp/build_grass
 mkdir "$TMP_DIR"
 
 apt-get --quiet update
-apt-get --yes install grass-core grass-gui grass-doc grass-dev libgdal20-2.1.0-grass
+apt-get --yes install grass-core grass-gui grass-doc grass-dev libgdal20-2.1.2-grass
```

Comment:
You should change `libgdal20-2.1.2-grass` to `libgdal-grass`, see: https://trac.osgeo.org/osgeo/ticket/1776